### PR TITLE
Add PHP method calls and misc small statements

### PR DESCRIPTION
**Pull request by johannescoetzee**
This also adds namespaces and some methodFullNames (with unresolved signatures)

### CODE REVIEW

File: joern-cli/frontends/php2cpg/src/main/scala/io/joern/php2cpg/parser/Domain.scala [hunk 1]
```diff
@@ -576,7 +598,12 @@ object Domain {
 
   private def readVariable(json: Value): PhpVariable = {
     // TODO Figure out when the variable has an expr name
```
Comment (max-leuthaeuser) [inline joern-cli/frontends/php2cpg/src/main/scala/io/joern/php2cpg/parser/Domain.scala:600]: This TODO is done now?

**Review by max-leuthaeuser — APPROVED** [review]

File: joern-cli/frontends/php2cpg/src/main/scala/io/joern/php2cpg/parser/Domain.scala (hunk 1 above)
Comment (johannescoetzee) [reply joern-cli/frontends/php2cpg/src/main/scala/io/joern/php2cpg/parser/Domain.scala:600]: It is! Good catch

**Comment by fabsx00** [comment]
There seems to be something that changed about the macos-11 runner related to PHP.

**Comment by johannescoetzee** [comment]
It looks like it was just a network blip. A maven package fetch failed, but all's good on a rerun.

**Pull request merged**